spec/support/saml_response_helper.rb

```diff
@@ -34,6 +34,13 @@ def raw_xml_response
       end
     end
 
+    def saml_response(settings)
+      @_saml_response ||= OneLogin::RubySaml::Response.new(
+        raw_xml_response,
+        settings: settings
+      )
+    end
```

Comment:
I have completely refactored the SAML response helpers in https://github.com/18F/identity-idp/pull/329